src/qml/views/keyframes/Parameter.qml

```diff
@@ -30,6 +30,11 @@ Item {
 
     signal clicked(var keyframe, var parameter)
 
+    function setLowestHighest(lh) {
```

Comment:
Name suggestions:
Change "setLowestHighest" to "setMinMax"
Change "lh" to "zoomHeight"
thoughts?

Reply:
Yeah, I have no problem with changing names.
Updated with two commits below.